hack/local-up-cluster.sh

```diff
@@ -161,7 +161,8 @@ function usage {
 # This function guesses where the existing cached binary build is for the `-O`
 # flag
 function guess_built_binary_path {
-  local hyperkube_path=$(kube::util::find-binary "hyperkube")
+  local hyperkube_path
+  hyperkube_path=$(kube::util::find-binary "hyperkube")
```

Comment:
In ./hack/local-up-cluster.sh line 164:
  local hyperkube_path=$(kube::util::find-binary "hyperkube")
        ^------------^ SC2155: Declare and assign separately to avoid masking return values.